test/test_transforms_v2_consistency.py

```diff
@@ -514,23 +395,6 @@ def __init__(
 ]
 
 
-def test_automatic_coverage():
```

Comment:
This was useful in the beginning, to ensure we are not missing something. But now that we start to remove configs here, we need to remove this check as well.